nss-day-cohort-60/rare-python-server-tyrannical-tators
# updated models, added repository.py in views for get all and get sing…

…le functions, updated main module with do_get function, mapper, and getall/getsingle function. Getter functions now working.

# Closes #42 # Closes #44

# Description 

Added category, comment, post, tag, and user models
Added repository.py in views and created get all functions for categories, posts, tags, and users, get single functions for posts & users
Added method mapper, get all/single, and do get functions to request_handler.py

# Tests
1. Fetch this branch to your machine
2. Open postman
3. Open code
4. Send GET request to http://localhost:8088/categories, http://localhost:8088/tags, http://localhost:8088/posts, http://localhost:8088/users
5. Make sure each request returns a list of dictionaries as JSON strings and status code 200
6. Send GET request to http://localhost:8088/users/1 and http://localhost:8088/posts/1
7. Make sure each request returns a JSON stringified dictionary and status code 200
8. Send GET request to http://localhost:8088/users/100 and http://localhost:8088/posts/100
9. Each request should return an empty string and 404 error
10. Send GET request to http://localhost:8088/schmoopity
11. Request should return an empty string and 404 error
